tests/_support/AcceptanceTester.php

```diff
@@ -23,4 +23,39 @@ class AcceptanceTester extends \Codeception\Actor
    /**
     * Define custom actions here
     */
+
+   	/**
+	 * Sets tickets enabled post types
+	 *
+	 * @param  array $post_types The post types to be enabled for tickets. Can be empty.
+	 */
+	public function haveTicketablePostTypes ( array $post_types = array( ) ) {
+
+		$I = $this;
```

Comment:
you don't _need_ to change this, but I'd recommend just using `$this` all of the time instead of aliasing it to `$I`

although if this is a common (or even recommended) pattern with acceptance tests, which it seems to be, then please ignore this comment 😎

Reply:
@ggwicz this seems to be the good practice, found it this way in the official codeception docs.